web_developer_mode_optional/static/src/js/chrome.js

```diff
@@ -0,0 +1,43 @@
+// License, author and contributors information in:
+// __openerp__.py file at the root folder of this module.
+
+/*---------------------------------------------------------
+ * OpenERP Web chrome
+ *---------------------------------------------------------*/
+openerp.web_developer_mode_optional = function () {
+    var instance = openerp;
+    openerp.web.chrome = {};
+    var QWeb = instance.web.qweb;
+    var _t = instance.web._t;
+
+    instance.web.UserMenuDevel =  instance.web.UserMenu.include({
+        on_menu_about: function() {
```

Comment:
probably you're better off doing your stuff in an override of the qweb view, and only set you flag in js